### PR TITLE
category listing UI

### PR DESCRIPTION
**Pull request by vidhya0617**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -10,4 +12,16 @@ module.exports = {
       "secure.gravatar.com",
     ],
   },
+  async rewrites() {
+    return [
+      {
+        source: "/learn/:slug([a-z].*)*",
+        destination: "/category/:slug*",
```
Comment (ArunAnthony) [inline next.config.js:19]: what would be the URL for this ? will it still be /learn/baby-health ?? or  /category/baby-health?

File: src/components/common/CategoryCard.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable  */
+
+import React from "react";
+import Image from "next/image";
+
+const CategoryCard = (props) => {
+  const { customClassName } = props;
+  return (
+    <div className={`py-3 px-2.5 float-left cursor-pointer w-full xl:w-1/3 lg:w-1/2 sm:w-1/2  ${customClassName}`}>
```
Comment (ArunAnthony) [inline src/components/common/CategoryCard.tsx:9]: Aren't we using the feedpost card you created for article detail page ??

File: next.config.js (hunk 1 above)
Comment (vidhya-balasubramanian) [reply next.config.js:19]: It will be learn/baby- health.

Comment (vidhya-balasubramanian) [reply next.config.js:19]: Source is expected route and Destination is folder path

**Pull request merged**